gulpfile.js

```diff
@@ -465,7 +468,21 @@ function deployCesium(bucketName, uploadDirectory, cacheControl) {
         })
         .then(function() {
             console.log('Cleaned ' + existingBlobs.length + ' files.');
+        })
+        .catch(function(error) {
+            errors.push(error);
         });
+    }).then(function() {
+        if (errors.length === 0) {
+            done();
+            return;
+        }
+
+        console.log('Errors: ');
+        errors.map(function(e) {
+            console.log(e);
+        });
+        done(1);
```

Comment:
I've bikeshedded enough so there's no need to change this, but I'm pretty sure you could have just returned `Promise.reject()` here instead of going pack to using the done parameter.